Nos/Views/ProfileEdit/CreateUsernameWizard/ClaimYourUniqueIdentitySheet.swift

```diff
@@ -15,7 +15,16 @@ struct ClaimYourUniqueIdentitySheet: View {
                 PickYourUsernameSheet(isPresented: $isPresented)
             }
             .buttonStyle(BigActionButtonStyle())
-            Spacer(minLength: 40)
+            NavigationLink {
+                AlreadyHaveANIP05View(isPresented: $isPresented)
+            } label: {
+                SwiftUI.Text(.localizable.alreadyHaveANIP05)
+                    .font(.clarity(.medium, textStyle: .footnote))
```

Comment:
I think we should bump this font up a size? The text size looks smaller in relation to the button above it than it does in Figma.

Reply:
In Figma it's 14. Looking at https://developer.apple.com/design/human-interface-guidelines/typography: footnote is 13 and subheadline is 15. That's why I suggested the designs specify the dynamic type instead. I'll change it to subheadline because it's ambiguous.

Reply:
looks much better to my eyes now. Thanks!